### ISSUE TITLE
Lost Pet

### ISSUE THREAD

**Issue by NathonS** [elysium-project/bug-tracker#184]
Nighthaven PVP (1.2)

Was walking back to lights hope chapel and noticed pet was following so i tried calling pet and said i had no pet, I'm a hunter i feed him every time i call him and every time he dies he was at max loyalty and happiness so i checked combat log and nothing showed up, GM said to contact this site

**Comment by Terranzin** [comment]
We can´t assist you with bringing the pet back. We can look into the problem, but that won´t solve your problem for now, instead you´ll need to either find a new pet or ask the GM team to assist you. Could you please provide more information how you lost him in the first place?

**Comment by Terranzin** [comment]
It seems to be a db issue. There have been characters every now and then that are bugged and have to be moved to new account ID in the database. Please contact our ingame support (make a ticket) and ask them to get help from the administration.

**Comment by NathonS** [comment]
Happened pretty quick landed in lights hope chapel pet was dead so I
revived him and fed him to max happiness. I grinded a few mobs and got an
invite to a dungeon I hopped on my mount and went towards my group waiting
for me. On the way I was dazed by a mob that I tried to ride past so I
hopped off my mount and told my pet to attack when he didn't respond I
checked and saw that he wasn't there I guessed that he to had been dazed at
some time and was dismissed for being to far so I attempted to whistle for
him, when that didn't work I feign death so mob would stop attack and
hastily went to lights hope chapel to find stable master as was hoping he
glitched and was returned to stable. Couldn't find a stable master so I
hearthed to my home in at the qetlands zone before I check stable master I
gave one more try to whistle it was at this point red letters came across
my screen informing me I didn't have a pet and as such couldn't whistle for
his return. In a panic I went to stable master and he to didnt have my pet
it was at this point I realized I was screwed....

In shorr I contactwd GMs who told me to contact bug tracker as they cant
assist with lost pet... You guys informed me you as well wouldnt be able to
help me gwt my pet back.... Sad day in the life of a hunter as my pet is no
longer able to be obtained without multiple days of sitting and waiting for
him with no guarantee I will even be able to tame him once he spawned.. I
still hold hope that something can be done but begin to have doubts as to
wether or not an attempt was tried no one gave me a character recovery
option instead saying good luck finding a new pet. I guess this way only
considered to be a problem by me....
